modules/reference/pages/cluster-properties.adoc

```diff
@@ -293,6 +293,46 @@ Max number of simultaneous connections to S3 per shard. Includes connections use
 
 ---
 
+=== cloud_storage_recovery_topic_validation_mode
+
+Validation mode performed before recovering a topic from cloud storage. This property accept the following parameters:
+
+- `no_check`. Skips the checks for topic recovery.
+- `check_manifest_existence`.  Run an existence check of each `partition_manifest`. Fails if there are connection issues to the cloud storage.
+- `check_manifest_and_segment_metadata`. Download the manifest and run a consistency check, comparing the metadata with the cloud storage objects. The process will fail if any metadata references cloud storage objects that are missing.
```

Comment:
@andijcr Does this outputs which objects are missing?

Reply:
in case of failure, the reason for failure will appear as ERROR lines in the log of redpanda

for each topic, it will report errors for all the partitions, but for each partition, only the first error is reported.

let me prepare a table to explain this

Reply:
Which log? How can the user access this log?

Reply:
> Which log
redpanda application log.
> How can the user access this log
depends on the deployement

Reply:
Do you have an example of an error line?

Reply:
here's an example
```
ERROR 2024-04-24 21:29:08,166 [shard 1:main] cluster - [fiber11|0|299996ms recovery validation of {kafka/panda-topic-recovery-NOT-OK/0}/24] - manifest metadata check: missing segment, validation not ok
ERROR 2024-04-24 21:29:08,166 [shard 1:main] cluster - topics_frontend.cc:519 - Stopping recovery of {kafka/panda-topic-recovery-NOT-OK} due to validation error
```

it's validating topic `kafka/panda-topic-recovery-NOT-OK` , and it's stopping due to a fatal error on partition 0

the first line is ERROR about the error, has the form
```
ERROR .... [... recovery validation of {<namespace/topic/partition>}...] - <failure reason>, validation not ok
```
and for each failing partition there will be an ERROR message like this.
at the end of the process, there will be a  final ERROR message

```
ERROR ... ... - Stopping recovery of {<namespace/topic>} due to validation error
```